unfurl/job.py

```diff
@@ -35,7 +35,7 @@
 from .localenv import LocalEnv
 
 # note: need to import configurators even though it is unused
-from . import display, init_logging, configurators
+from . import display
```

Comment:
notice the comment above that import statement: "note: need to import configurators even though it is unused" -- you deleted that import, please put it back

Reply:
fixed